servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -4122,19 +4109,21 @@ RendererSceneRenderRD::RendererSceneRenderRD(RendererStorageRD *p_storage) {
 
 	uint32_t textures_per_stage = RD::get_singleton()->limit_get(RD::LIMIT_MAX_TEXTURES_PER_SHADER_STAGE);
 
+	/* Removing low end from here. Leaving this commented to discuss with Reduz, should we make sure that if textures_per_stage is low we switch to mobile renderer? 
```

Comment:
So we were switching low end on hardcoded if `textures_per_stage` was less then 48, should we move this code into the mobile renderer PR and force the high end renderer to be ignored if this value is too low?

Reply:
I think so. We need @reduz' confirmation, but the way I see it is that we will have a low-end and a high-end renderer, rather than desktop/mobile. 

Low end will be optimized for mobile / integrated chips while high end will be for dedicated GPUs

Reply:
Yeah thats a good point, we've been calling it the mobile renderer but maybe thats not the best name for it if it also becomes the stand in for low end.

Reply:
Ok this check is now in the mobile renderer PR:
```
	String mode = GLOBAL_DEF("rendering/vulkan/rendering/mode", "Forward clustered");
	ProjectSettings::get_singleton()->set_custom_property_info("rendering/vulkan/rendering/mode",
			PropertyInfo(Variant::STRING,
					"rendering/vulkan/rendering/mode",
					PROPERTY_HINT_ENUM, "Forward clustered,Forward mobile"));

	uint32_t textures_per_stage = RD::get_singleton()->limit_get(RD::LIMIT_MAX_TEXTURES_PER_SHADER_STAGE);

	if (mode == "Forward mobile" || textures_per_stage < 48) {
		scene = memnew(RendererSceneRenderImplementation::RenderForwardMobile(storage));
	} else {
		// default to our high end renderer
		scene = memnew(RendererSceneRenderImplementation::RenderForwardClustered(storage));
	}
```